.gitignore

```diff
@@ -1,6 +1,13 @@
 .DS_Store
+.idea
+.vscode
```

Comment:
I want to have the `.vscode` folder in the git repo because it contains the debug configurations.